Blockzilla/Menu/Protocol/MenuActionable.swift

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+protocol MenuActionable: AnyObject {
```

Comment:
The elements in this menu can not be accessed with IDs now. And their tree structure has changed to. For example to access Share Page With.. menu, before it was like:
`app.cells["Share Page With..."]`
And now:
`app.cells.buttons["Share Page With..."]`

Reply:
Indeed @isabelrios, the difference comes from being separate classes now. Good catch. I'm wondering how to handle this, we only use the native component from iOS14 onwards. Can we conditionally test?
@st3fan

Reply:
Interesting fact @isabelrios , `UIMenu` uses a `collection view` so you would call the buttons with `app.collectionViews.cells.buttons["Share Page With..."]`